### ISSUE TITLE
Better ban message

### ISSUE THREAD

**Issue by omerxx** [omerxx/fatbot#49]
Message is hard to follow, users end up sending the bot in private.
Text not aligned

**Comment by Ofirham** [comment]
`%s You were banned from the group after not working out.
You can rejoin after 48 hours:
1. Send here (or press on the word) /join
2. Wait for the admin approval
3. Get a link to join the group

** NOTICE!! After joining the group, you will have 60 minutes to send 2 workouts (pictures) in the group chat.`,
